DebugSwift/Sources/Features/App/App.Controller.swift

```diff
@@ -159,7 +159,16 @@ extension AppViewController: UITableViewDataSource, UITableViewDelegate {
             case .crash:
                 let controller = CrashViewController()
                 navigationController?.pushViewController(controller, animated: true)
+            case .simulatePush:
+                    if #available(iOS 13.0, *) {
+                        let controller = LocalNotificationSimulatorViewController()
+                        navigationController?.pushViewController(controller, animated: true)
+                    } else {
+                        // no fallback
```

Comment:
need create some alert here

Reply:
it shouldn't actually.  in `allCasesWithPermission`, the simulate push will be removed if they are not on a compatible version 

I'll remove the `else` here, i think that should be enough